artifacts/compatibility-yaml/compatibility-v1.0.0.yaml

```diff
@@ -0,0 +1,65 @@
+{
+  "CSI" : {
+    "3.0.0" : {
+      "vSphere": { "min": "6.7.1", "max": "8.2.0" },
+      "k8s": { "min": "1.25", "max": "1.27" },
+      "isCPIRequired": false,
+      "deploymentPath": [
+          "https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/release/artifacts/csi/3.0.0/namespace.yaml",
+          "https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/release/artifacts/csi/webhook.yaml",
+          "https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/release/artifacts/csi/3.0.0/vsphere-csi-driver.yaml" ]
+    },
+    "2.7.0" : {
+      "vSphere": { "min": "6.7.1", "max": "8.0.1" },
+      "k8s": { "min": "1.23", "max": "1.25" },
+      "isCPIRequired": false,
+      "deploymentPath": [
+          "https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/release/artifacts/csi/namespace.yaml",
+          "https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/release/artifacts/csi/webhook.yaml",
+          "https://raw.githubusercontent.com/vmware-tanzu/vsphere-kubernetes-drivers-operator/release/artifacts/csi/vsphere-csi-driver.yaml" ]
+    },
+    "2.6.2": {
+      "vSphere": { "min": "6.7.1", "max": "8.0.1" },
+      "k8s": { "min": "1.22", "max": "1.24" },
+      "isCPIRequired": false,
+      "deploymentPath": [
+          "https://raw.githubusercontent.com/kubernetes-sigs/vsphere-csi-driver/v2.6.2/manifests/vanilla/namespace.yaml",
```

Comment:
Why don't we need the webhook.yaml or validatingwebhook.yaml for CSI 2.6.2, similar to other versions?

Reply:
Because this was introduced from 2.7 onwards